gui-js/apps/minsky-electron/src/app/managers/ContextMenuManager.ts

```diff
@@ -189,20 +187,55 @@ export class ContextMenuManager {
       );
     }
   }
+
+  private static async leftMouseGodley(mainWindow: BrowserWindow) {
+    try {
+      const itemInfo = await CommandsManager.getItemInfo(this.x, this.y);
+      if(itemInfo?.classType === ClassType.GodleyIcon) {
+        let godley=new GodleyIcon(minsky.canvas.item);
+        minsky.nameCurrentItem(await godley.id());
+        const editorModeChecked = await godley.editorMode();
+
+        const editorX = await godley.toEditorX(this.x) / await godley.editor.zoomFactor();
+        const editorY = await godley.toEditorY(this.y) / await godley.editor.zoomFactor();
```

Comment:
Probably worth pulling the constant await zoomfactor call in front of the toEditor calls.